torch/_subclasses/fake_tensor.py

```diff
@@ -860,13 +860,24 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         # and do device logic, we dont need do anything but run them
         # and ensure that Meta kernels are dispatched to (see)
         # Fake Tensor Dispatch Keys
+        if (
+            "prims::" in func._schema.name
+            and len(flat_arg_fake_tensors) != 0
+            and not hasattr(func, "prim_meta_impl")
+        ):
+            raise RuntimeError(
```

Comment:
This seems to be disagreeing with the fallback. Should we raise runtime error here or go with the fallback as
```
return run_fallback_kernel(self, func, args, kwargs, not_implemented_error)
```